LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ReceivedLoRaCloudToDeviceMessage.cs

```diff
@@ -36,10 +35,9 @@ public byte[] GetPayload()
                 {
                     return Convert.FromBase64String(RawPayload);
                 }
-                catch (FormatException ex)
+                catch (FormatException)
                 {
                     // Invalid base64 string, return empty payload
-                    Logger.Log($"Payload '{RawPayload}' is not a valid base64 value: {ex.Message}", LogLevel.Error);
```

Comment:
Logging removed (not sure why) - should we use here as well the `ExceptionFilterUtility` ?

Reply:
Valid point. This is one of the few logging statements that I actively removed - I thought it wasn't worth the hassle to inject a logger from all call sites into this method. Tagging @Mandur to see whether I should bring this back or whether it's ok to resolve.

Reply:
Hmmm imho this looks like an important log as it reacts on malformed input by the user. I was wondering about throwing catch it locally to abandon the message and rethrow, catching on the global filter to log the exception.

**I would anyway write an issue and tackle it in a different work item to be able to calmly do that**

Reply:
Created #857 for this.